resources/home/dnanexus/eggd_artemis.py

```
@@ -0,0 +1,438 @@
+"""Main app entrypoint"""
```

Ruff (0.8.0)

The file is executable but no shebang is present

(EXE002)